src/html/modules/report/about.html

```diff
@@ -0,0 +1,112 @@
+<div class="container">
```

Comment:
1) `src/html/modules/report/about.html` is not correct name for the file's purpose. It contains the form, so I would name it `form.html`
2) If you take a look at other pages and files, you will see that we use more precise word for naming the root class in the file (not `container`)